### PR TITLE
[DEVOPS-951] Test connection avalablity in global configuration page

### PR DESCRIPTION
**Pull request by cizezsy**

### CODE REVIEW

File: src/main/resources/com/alauda/jenkins/plugins/ClusterConfig/config.jelly [hunk 1]
```diff
@@ -26,4 +26,7 @@
         <f:textbox/>
     </f:entry>
 
+    <f:validateButton title="${%Test Connection}" method="verifyConnect"
```
Comment (LinuxSuRen) [inline src/main/resources/com/alauda/jenkins/plugins/ClusterConfig/config.jelly]: 在这里加一下本地化 https://github.com/jenkinsci/localization-zh-cn-plugin

Comment (cizezsy) [reply src/main/resources/com/alauda/jenkins/plugins/ClusterConfig/config.jelly]: OK

Comment (cizezsy) [reply src/main/resources/com/alauda/jenkins/plugins/ClusterConfig/config.jelly]: @LinuxSuRen 加了 :P

**Comment by jiaozhang1** [comment]
pass test

**Pull request merged**